test/e2e/utils_test.go

```diff
@@ -183,6 +183,13 @@ func execCmdOnNode(t *testing.T, cs *framework.ClientSet, node corev1.Node, args
 	return string(b)
 }
 
+func getOsReleaseFromNode(t *testing.T, cs *framework.ClientSet, node corev1.Node) (string, string) {
+	osId := execCmdOnNode(t, cs, node, "bash", "-c", "cat /rootfs/etc/os-release | grep '^ID=' | sed -e 's/^ID=//'")
```

Comment:
This is going to be fragile around shell quoting handling.  The `os-release` files are designed instead to be sourced by the shell.  So what you really want to do is more like
```
bash -c ". /rootfs/etc/os-release; echo $ID"
```